crates/solidity/testing/snapshots/cst_output/TryStatement/try_catch/generated/0.4.11-failure.yml

```diff
@@ -0,0 +1,17 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
```

Comment:
> we lose the ability to test it as a named parser

maybe we can test the parent `StatementsList` or `Block` instead?

Reply:
Sure, this also works.